powerauth-load-tests/src/test/java/com/wultra/security/powerauth/test/config/PowerAuthLoadTestCommon.java

```diff
@@ -0,0 +1,190 @@
+ * <p>
+ * All configuration values can be customized via environment variables. Defaults are provided for ease of use
+ * and quick setup.
+ * </p>
+ *
+ * @author Jan Dusil, jan.dusil@wultra.com
+ */
+@Slf4j
+public class PowerAuthLoadTestCommon {
+
+    @Setter
+    @Getter
+    private static boolean isPrep = false;
+
+    private static FeederBuilder<Object> userDataFeedList;
+
+    private static FeederBuilder.FileBased<Object> userDataFeedJson;
+
+    private static final String DATA_DUMP_FILE = "registrationDataFeed.json";
+
+    /* Empirical value from testing */
+    public static final int MAX_CONCURRENT_USERS = 10;
+
+    /* Total number of registrations to prepare during preparation phase */
+    public static final int PERF_TEST_PREP_N_REG = getIntEnv("PERF_TEST_PREP_N_REG", 100);
+
+    /* Total number of operations per registration to prepare during preparation phase */
+    public static final int PERF_TEST_PREP_M_OP = getIntEnv("PERF_TEST_PREP_M_OP", 1000);
+
+    /* Number of registrations to add during execution phase per sec */
+    public static final int PERF_TEST_EXE_X_REG = getIntEnv("PERF_TEST_EXE_X_REG", 1);
+
+    /* Number of operations to add during execution phase per sec */
+    public static final int PERF_TEST_EXE_Y_OP = getIntEnv("PERF_TEST_EXE_Y_OP", 50);
+
+    /* The length of the execution phase in minutes */
+    public static final int PERF_TEST_EXE_MIN = getIntEnv("PERF_TEST_EXE_MIN", 1);
+
+    public static final String PAC_ADMIN_USER = getStringEnv("PAC_ADMIN_USER", "system-admin");
+    public static final String PAC_ADMIN_PASS = getStringEnv("PAC_ADMIN_PASS", "");
+    public static final String PAC_URL = getStringEnv("PAC_URL", "http://localhost:8089/powerauth-cloud");
+    public static final String TEST_SERVER_URL = getStringEnv("TEST_SERVER_URL", "http://localhost:8081");
+    public static final String CALLBACK_URL = getStringEnv("PERF_TEST_CALLBACK_URL", "http://localhost:8090/mock-callback");
+
+    /**
+     * Common HTTP protocol configuration for Gatling tests.
+     */
+    public final static HttpProtocolBuilder commonProtocol = http
+            .contentTypeHeader("application/json")
+            .acceptHeader("application/json")
+            .userAgentHeader("PowerAuth-LoadTest/gatling").check();
+
+    /**
+     * Fetches an integer environment variable by name, or returns the default value if not found or invalid.
+     *
+     * @param name         The name of the environment variable.
+     * @param defaultValue The default value to use if the environment variable is not found or invalid.
+     * @return The environment variable value as an integer, or the default value.
+     */
+    private static int getIntEnv(final String name, final int defaultValue) {
+        final String value = System.getenv(name);
+        if (value != null && !value.isEmpty()) {
+            try {
+                return Integer.parseInt(value);
+            } catch (NumberFormatException e) {
```

Comment:
Since Apache Commons already on the classpath, you may use `NumberUtils#toInt(String, int)`

Reply:
Honestly, I like this better because it gives me chance to log for the user that the default value is used instead of the one in the env file.